worker.Dockerfile

```diff
@@ -3,26 +3,13 @@ WORKDIR /go/src/subtrace
 COPY . .
 RUN --mount=type=cache,target=/go/pkg --mount=type=cache,target=/root/.cache make
 
-FROM debian:12
-
-RUN apt-get update && apt-get install -y ca-certificates curl
-
-ARG TARGETARCH
-RUN cat | bash - <<EOF
-  curl -sLO "https://packages.clickhouse.com/deb/pool/main/c/clickhouse/clickhouse-common-static_23.12.6.19_${TARGETARCH}.deb"
-  curl -sLO "https://packages.clickhouse.com/deb/pool/main/c/clickhouse/clickhouse-server_23.12.6.19_${TARGETARCH}.deb"
-  DEBIAN_FRONTEND=noninteractive apt install -y ./clickhouse-common-static_23.12.6.19_${TARGETARCH}.deb ./clickhouse-server_23.12.6.19_${TARGETARCH}.deb
-  rm ./clickhouse-common-static_23.12.6.19_${TARGETARCH}.deb ./clickhouse-server_23.12.6.19_${TARGETARCH}.deb
-EOF
-
+FROM clickhouse/clickhouse-server:24.8-alpine
```

Comment:
Sweet! We're at the mercy of the CH image now but it should be fine, I'd rather they maintain this stuff than us trying to do something custom

Reply:
Even previously, we were using the official build with instructions from their website. Not much has changed other than the fact that our Dockerfile is shorter and network blob download caching is easier.